<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="health">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/pics/health.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="attack">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/pics/sword.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="armor">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/pics/shield.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="story">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>371</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#story{
	color: #FFFFFF;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shop">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#shop{
border-image: url(:/pics/items.png);
}
#shop:hover{
border-image: url(:/pics/gold.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gold">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/pics/goldpot.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="attackIncrease">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>510</y>
      <width>681</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#attackIncrease{
	background-color:#c62d1f;
	border-radius:8px;
	color:#ffffff;
	font-size:17px;
	text-decoration:none;
}
#attackIncrease:hover {
	background-color:#f24437;
}</string>
    </property>
    <property name="text">
     <string>Upgrade Attack</string>
    </property>
   </widget>
   <widget class="QPushButton" name="defenseIncrease">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>540</y>
      <width>681</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#defenseIncrease{
	background-color:#4d79ff;
	border-radius:8px;
	color:#ffffff;
	font-size:17px;
	text-decoration:none;
}
#defenseIncrease:hover {
	background-color:#809fff;
}</string>
    </property>
    <property name="text">
     <string>Upgrade Defense</string>
    </property>
   </widget>
   <widget class="QPushButton" name="healthIncrease">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>570</y>
      <width>681</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#healthIncrease{
	background-color:#53ff1a;
	border-radius:8px;
	color:#ffffff;
	font-size:17px;
	text-decoration:none;
}
#healthIncrease:hover {
	background-color:#8cff66;
}</string>
    </property>
    <property name="text">
     <string>Upgrade Health</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restore">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>480</y>
      <width>681</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#restore{
	background-color:#666699;
	border-radius:8px;
	color:#ffffff;
	font-size:17px;
	text-decoration:none;
}
#restore:hover {
	background-color:#8585ad;
}</string>
    </property>
    <property name="text">
     <string>Restore Health</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcome">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>149</y>
      <width>1051</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#welcome{
	color:#FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="forest">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#forest{
	border-image: url(:/pics/forest.png);
}
#forest:hover{
	border-image: url(:/pics/goblin.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="goldAmount">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#goldAmount{
	color:#FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="healthAmount">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>130</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#healthAmount{
	color:#FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="attackAmount">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#attackAmount{
	color:#FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="defenseAmount">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#defenseAmount{
	color:#FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="notifications">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#notifications{
	color:#FFFFFF;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>160</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#save{
	background-color:#89c403;
	border-radius:6px;
	color:#ffffff;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
	padding:6px 24px;
}
#save:hover {
	background-color:#77a809;
}
</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#load{
	background-color:#4d79ff;
	border-radius:6px;
	color:#ffffff;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
	padding:6px 24px;
}
#load:hover {
	background-color:#809fff;
}
</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="playerName">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#playerName{
	color:#FFFFFF;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="player">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>290</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#player{
	color:#FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>Enter name below to save/load file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dragon">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#dragon{
	border-image: url(:/pics/dragon.png)
}
#dragon:hover{
	border-image: url(:/pics/exclamation.png)
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>attack</zorder>
   <zorder>health</zorder>
   <zorder>armor</zorder>
   <zorder>story</zorder>
   <zorder>shop</zorder>
   <zorder>gold</zorder>
   <zorder>attackIncrease</zorder>
   <zorder>defenseIncrease</zorder>
   <zorder>healthIncrease</zorder>
   <zorder>restore</zorder>
   <zorder>welcome</zorder>
   <zorder>forest</zorder>
   <zorder>goldAmount</zorder>
   <zorder>healthAmount</zorder>
   <zorder>attackAmount</zorder>
   <zorder>defenseAmount</zorder>
   <zorder>notifications</zorder>
   <zorder>save</zorder>
   <zorder>load</zorder>
   <zorder>playerName</zorder>
   <zorder>player</zorder>
   <zorder>dragon</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
